lib/api/v2/search_as_json.rb

```diff
@@ -32,17 +32,6 @@ def as_json_results_to_hash
     @results.collect { |result| as_json_result_hash result }
   end
 
-  def as_json_result_hash(result)
-    result.description ||= result.content
-    result.url ||= result.unescaped_url
-    {
-      title: result.title,
-      url: result.url,
-      display_url: result.display_url,
-      snippet: as_json_build_snippet(result.description),
-    }
-  end
-
```

Comment:
Similar comment as above. This may be expendable, but without full API integration tests (or my own code crawl), I'm nervous about removing it. Let's restore it, at least for now.